Sources/WebDriver/Requests.swift

```diff
@@ -531,4 +531,18 @@ public enum Requests {
 
         public typealias Response = WebDriverStatus
     }
+
+    public struct SessionWindowHandle: Request {
+        public var session: String 
+        public var many: Bool
+
+        public var pathComponents: [String] { ["session", session, many ? "window_handles" : "window_handle"] }
+        public var method: HTTPMethod { .get }
+
+        public typealias Response = ResponseWithValue<ResponseValue>
+        public struct ResponseValue: Codable {
```

Comment:
The spec says that the response value is a string/array rather than a struct containing a string property. The correct definition should be:

```
public typealias Response = ResponseWithValue<String>
```
(and an array for the handles variant)